### PR TITLE
google machine learning api

### PR DESCRIPTION
**Pull request by PratikDhanave**
google machine learning api
CreateMLModel()
DeleteMLModel()
GetMLModel()
UpdateMLModel()
Apidoc
example
docs

### CODE REVIEW

File: machinelearning/googlemachinelearning/googlemachinelearning.go [hunk 1]
```diff
@@ -1,25 +1,352 @@
...
+							option.defaultVersion.manualScaling.nodes = nodesv
+						}
+					}
+				}
+			}
+		}
+	}
+
+	createMLModeljsonmap := make(map[string]interface{})
+
```
Comment (rehrumesh) [inline machinelearning/googlemachinelearning/googlemachinelearning.go]: remove unnecessary line breaks.

File: machinelearning/googlemachinelearning/googlemachinelearning.go [hunk 2]
```diff
@@ -1,25 +1,352 @@
...
+					}
+				}
+			}
+		}
+	}
+
+	updateMLModeljsonmap := make(map[string]interface{})
+
+	createMLModeldictnoaryconvert(option, updateMLModeljsonmap)
+
```
Comment (rehrumesh) [inline machinelearning/googlemachinelearning/googlemachinelearning.go]: remove unnecessary line breaks.

**Comment by OddCN** [comment]
hi @PratikDhanave looks there are conflicts

**Comment by PratikDhanave** [comment]
@OddCN @rehrumesh  I will updated changes in new pull request

**Pull request closed without merging**